docs/2.guide/2.directory-structure/1.shared.md

```diff
@@ -15,6 +15,12 @@ The `shared/` directory is available in Nuxt v3.14+.
 Code in the `shared/` directory cannot import any Vue or Nitro code.
 ::
 
+::warning
+Auto-imports are not enabled by default in Nuxt v3 to prevent breaking changes in existing projects.
```

Comment:
`Auto-imports are not enabled by default in Nuxt v3` 
It's not by default. We don't enable it for V3

Reply:
you can enable them in v3 by setting compatibilityVersion.

Reply:
Oh I get it now. IMO it's a bit hard to draw a clear line between feature available in comp v4 and those in the "standard" configuration when saying that it can be enabled in V3.
We can change it later if some misunderstand this sentence.